checks/devnet/bootstrap/D_shard_0(0)_epoch_1166_nonce_1399025_XeGLD.rosetta.json

```diff
@@ -0,0 +1,1162 @@
+[
```

Comment:
can not check these values

Reply:
Balances bootstrapping files were generated using [balancesExporter](https://github.com/ElrondNetwork/elrond-tools-go/tree/main/trieTools/balancesExporter). The flow is described in the readme file. However, they can also be checked based on the existing bootstrap files:

```
package main

import (
	"encoding/json"
	"fmt"
	"io/ioutil"
	"reflect"

	"github.com/ElrondNetwork/elrond-go-core/core/pubkeyConverter"
	logger "github.com/ElrondNetwork/elrond-go-logger"
)

type rosettaBalance struct {
	AccountIdentifier rosettaAccountIdentifier `json:"account_identifier"`
	Currency          *rosettaCurrency         `json:"currency"`
	Value             string                   `json:"value"`
}

type rosettaAccountIdentifier struct {
	Address string `json:"address"`
}

type rosettaCurrency struct {
	Symbol   string `json:"symbol"`
	Decimals uint   `json:"decimals"`
}

func main() {
	projectedShard := byte(0)
	addressConverter, _ := pubkeyConverter.NewBech32PubkeyConverter(32, logger.GetOrCreate(""))
	dataWithActualShard, _ := ioutil.ReadFile("D_shard_0_epoch_1166_nonce_1399025_XeGLD.rosetta.json")
	dataWithProjectedShard, _ := ioutil.ReadFile("D_shard_0(0)_epoch_1166_nonce_1399025_XeGLD.rosetta.json")

	recordsWithActualShard := make([]rosettaBalance, 0)
	recordsWithProjectedShard := make([]rosettaBalance, 0)
	recordsWithProjectedShardRebuilt := make([]rosettaBalance, 0)
	_ = json.Unmarshal(dataWithActualShard, &recordsWithActualShard)
	_ = json.Unmarshal(dataWithProjectedShard, &recordsWithProjectedShard)

	for _, record := range recordsWithActualShard {
		publicKey, _ := addressConverter.Decode(record.AccountIdentifier.Address)
		if publicKey[len(publicKey)-1] == projectedShard {
			fmt.Println(record.AccountIdentifier.Address, record.Value)
			recordsWithProjectedShardRebuilt = append(recordsWithProjectedShardRebuilt, record)
		}
	}

	fmt.Println("Equal?", reflect.DeepEqual(recordsWithProjectedShardRebuilt, recordsWithProjectedShard))
}
```